src/drive/mobile/components/Settings.jsx

```diff
@@ -16,16 +16,16 @@ import Unlink from '../containers/settings/Unlink'
 
 class Settings extends Component {
   state = {
-    countClick: 0
+    tapCount: 0
```

Comment:
thanks for renaming this, it hurted my ears. ;-)

Reply:
Honestly, @y-lohse told me it would make you happy to do so, so I did :) but i didn't even think about it myself.